crates/tests/src/lib.rs

```diff
@@ -62,6 +62,7 @@ pub async fn render(scene: Scene, params: &TestParams) -> Result<Image> {
         RendererOptions {
             surface_format: None,
             use_cpu: params.use_cpu,
+            initialise_in_parallel: false,
```

Comment:
I wonder if it would be better to instead of a boolean control this setting via the number of threads,
such as  `num_initialisation_threads: Option<NonZeroUsize>`, with `None` using the heuristics, or
similar to rayon where instead it uses `usize` with `0 == heuristics` such as [rayon's `num_threads`](https://docs.rs/rayon/latest/rayon/struct.ThreadPoolBuilder.html#method.num_threads)

Reply:
Yeah, I think that's a good idea, thanks

Reply:
Could use futures underneath, have a default strategy (just fire up threads), and then have an `enum`/`Fn` interface for hooking in your own dispatch.

Reply:
@ratmice `Option<NonZeroUsize>` is guaranteed to be one word. :+ )

Reply:
Happy to resolve this following [aa0819e](https://github.com/linebender/vello/pull/455/commits/aa0819eb191ac96ff25ce26fa09b940ca2a50bcd), but leaving unresolved in case you want to tackle the futures aspect mentioned above under this too.